spec/features/dishes/show_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe 'Show Page', type: :feature do
+  describe 'As a visitor' do
+    before(:each) do
+      @chef_1 = Chef.create!(name: "Bobby")
+
+      @dish_1 = @chef_1.dishes.create!(name: "Slime", description: "Green Ooze")
+
+      @ingred_1 = Ingredient.create!(name: "Unknown", calories: 100)
+      @ingred_2 = Ingredient.create!(name: "Beans", calories: 60)
+      @ingred_3 = Ingredient.create!(name: "Mysterious Liquid", calories: 1000)
+
+      DishIngredient.create!(dish_id: @dish_1.id, ingredient_id: @ingred_1.id)
+      DishIngredient.create!(dish_id: @dish_1.id, ingredient_id: @ingred_3.id)
+    end
+    
+    # Us-1
+    it "displays the name of dish and it's description with the list of ingredients" do
+      # When I visit a dish's show page
+      visit dish_path(@dish_1)
+      # I see the dish’s name and description
+      expect(page).to have_content(@dish_1.name)
+      expect(page).to have_content(@dish_1.description)
+      # And I see a list of ingredients for that dish
+      within "#ingredient-#{@ingred_1.id}" do
+        expect(page).to have_content(@ingred_1.name)
```

Comment:
Nice work using within blocks! I'd argue they are not strictly necessary here because you don't have to test that any data besides the name is showing up. So, it's sufficient to test for the presence of the name. Within blocks could be really helpful here if you needed to also list the amount of calories under each ingredient. In that case, it would be really helpful to verify that "60" shows up in the beans section and "1000" shows up in the mysterious liquid section (great test data by the way). 

Just sharing so that you can focus on using within blocks when they are necessary for robust testing. It's not bad to use them here, just not as necessary!